chcemvediet/apps/accounts/migrations/0005_socialapp_remove_twitter_and_linkedin.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def forward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    SocialApp.objects.filter(pk__in=[3, 4]).delete()
+
+def backward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    twitter = SocialApp.objects.create(pk=3, provider=u'twitter', name=u'Twitter OAuth')
+    twitter.sites.add(1)
```

Comment:
Bolo by lepsie si tu site asi selectnut a potom pridat selectnutu site, nie iba rucne cislo. V migraciach by nemali byt explicitne zadane IDcka, ktore si vieme zistit.

Reply:
@martinmacko47 Ak som pridalval v migracii priamo objekt Site tak migracia nezbehla a vratilo obdobnu [chybu](https://stackoverflow.com/questions/29579871/django-data-migration-raising-a-type-error).  Cez shell mi to zbehlo v poriadku bez chyby. Sprava sa ti to rovnako? 
Pri pouziti priamo id-cka to prejde bez chyby.

```
site = Site.objects.get_current()
twitter = SocialApp.objects.create(provider=u'twitter', name=u'Twitter OAuth')
twitter.sites.add(site)  # error
twitter.sites.add(site.pk)  # ok 
```

Reply:
@viliambalaz Ano `site.pk` je OK. Ide o to, aby tam nebolo natvrdo cislo.

Reply:
Tak beriem spat. Predsa len budeme musiet pouzit `twitter.sites.add(SITE_ID)`. Vid komentar nizsie.